Source/WebCore/rendering/RenderTableCell.h

```diff
@@ -83,7 +83,7 @@ class RenderTableCell final : public RenderBlockFlow {
     void paint(PaintInfo&, const LayoutPoint&) override;
 
     void paintCollapsedBorders(PaintInfo&, const LayoutPoint&);
-    void paintBackgroundsBehindCell(PaintInfo&, const LayoutPoint&, RenderElement* backgroundObject);
+    void paintBackgroundsBehindCell(PaintInfo&, const LayoutPoint& paintOffset, RenderBox* backgroundObject, const LayoutPoint& backgroundPaintOffset);
```

Comment:
nit given the size of LayoutPoint, passing it by value is considered more performant.

Reply:
would you like me to change the existing LayoutPoint argument as well (it was a `const LayoutPoint&`)